### PR TITLE
New cli documentation

### PR DESCRIPTION
**Pull request by plopez26DPU**
Closes issue #218 

* [x] add CLI argument `build` documentation
* [x] add CLI understanding documentation
* [x] add accessing help menu
* [x] add running example 
* [x] take new screenshots
* [x] add all screenshots into documentation

### CODE REVIEW

**Comment by plopez26DPU** [comment]
Not sure what needs to be refactored, but it is concise as is.

**Review by Verisimilitude11 — CHANGES_REQUESTED** [review]
This looks great @plopez26DPU. Could you please clear the merge conflicts? This PR is first in our backlog, so I'll merge once you clear them. Thanks!

**Comment by plopez26DPU** [comment]
I think I fixed the issues @Verisimilitude11 , let me know if you have anymore issues :)

File: docs/samples/serine-dnalong.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
-    - `Main.java` is the entry point of the program.
-    - <img src="img/Screenshot 2022-08-24 123940.png" width="200"/>
-2. Run the program
-    - <img src="img/Screenshot 2022-08-24 124700.png" width="400"/>
\ No newline at end of file
+# CLI Arguments - Example
+
+## Build with Gradle
+
+**IMPORTANT**
```
Comment (Verisimilitude11) [inline docs/samples/serine-dnalong.md:5]: Label this with a title, for example, `OS-Specific Build Commands`

File: docs/samples/serine-dnalong.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+
+1. Type the build command in your terminal depending on your operating system.
+![](../usage/img/build.png)
+
+2. You should receieve the feedback in your terminal of the successful build.
+![](../usage/img/build2.png)
+
+## Standard CLI Understanding
+
+The CLI is consisted of two parts.
```
Comment (Verisimilitude11) [inline docs/samples/serine-dnalong.md:17]: The CLI consists of two parts.

File: docs/samples/serine-dnalong.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+```java
+java -jar build/libs/DNAnalyzer.jar <arguments>
+```
+
+To access the help menu type the following command into your terminal.
+```java
+java -jar build/libs/DNAnalyzer.jar -h
+```
+
+You will get the following response:
```
Comment (Verisimilitude11) [inline docs/samples/serine-dnalong.md:41]: You should get the following response:

File: docs/samples/serine-dnalong.md [hunk 4]
```diff
@@ -1,5 +1,57 @@
...
+```java
+java -jar build/libs/DNAnalyzer.jar <arguments>
+```
+
+Type the following into your terminal.
+```java
+java -jar build/libs/DNAnalyzer.jar assets/dna/random/dnalong.fa --amino=ser --min=14 --max=52
+```
+
+You will get the following response:
```
Comment (Verisimilitude11) [inline docs/samples/serine-dnalong.md:56]: You should get the following response:

File: docs/samples/serine-dnalong.md [hunk 5]
```diff
@@ -1,5 +1,57 @@
-1. Open `Main.java`
-    - `Main.java` is the entry point of the program.
-    - <img src="img/Screenshot 2022-08-24 123940.png" width="200"/>
-2. Run the program
-    - <img src="img/Screenshot 2022-08-24 124700.png" width="400"/>
\ No newline at end of file
+# CLI Arguments - Example
```
Comment (Verisimilitude11) [inline docs/samples/serine-dnalong.md:1]: Rename the file to this

**Review by Verisimilitude11 — CHANGES_REQUESTED** [review]
It looks excellent, @plopez26DPU! I added a couple of comments, but everything else looks fine. I'll merge once you complete that.

**Comment by plopez26DPU** [comment]
Check it out and let me know :)

**Comment by Verisimilitude11** [comment]
It looks fantastic, @plopez26DPU! By the way, would you like to join our Discord server to communicate more efficiently?

**Review by Verisimilitude11 — APPROVED** [review]

**Comment by plopez26DPU** [comment]
@Verisimilitude11 , sure ! That would be awesome . Do you have a invite link ?

**Comment by Verisimilitude11** [comment]
I posted a link in this discussion forum: https://github.com/Verisimilitude11/DNAnalyzer/discussions/256.

Congrats on completing Hacktoberfest!!! 🎉Will you still be interested in contributing after it ends? I greatly value your skill and expertise in DNAnalyzer, especially as we develop a front end and incorporate machine learning!

**Comment by plopez26DPU** [comment]
@Verisimilitude11 , of course I would like to continue to contribute to the project. I look forward to it

**Comment by Verisimilitude11** [comment]
Excellent; thank you so much for your continued support!

**Pull request merged**